tests/test_solveAll.py

```diff
@@ -28,8 +28,6 @@ def test_solveall_no_obj(self):
             else:
                 count = solver.solveAll(solution_limit=1000, display=add_sol)
             self.assertEqual(3, count)
-            self.assertSetEqual(sols,
```

Comment:
No no no, this check should remain! Ensur the `value` of Boolean variables is converted to True/False instead of `int`